doc/source/examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-cylinder-bed.rst

```diff
@@ -0,0 +1,312 @@
+==================================
+Gas-Solid Spouted Cylinder Bed
+==================================
+
+It is strongly recommended to visit `DEM parameters <../../../parameters/dem/dem.html>`_  and `CFD-DEM parameters <../../../parameters/unresolved-cfd-dem/unresolved-cfd-dem.html>`_ for more detailed information on the concepts and physical meaning of the parameters ind DEM and CFD-DEM.
+This is a modification of the example of `Gas-Solid Spouted Bed <../gas-solid-spouted-bed/gas-solid-spouted-bed.html>`_ . In this example, we deal with the geometry of a Cylinder. The boundary condition and some parameter have been changed accordingly.
+
+
+----------------------------------
+Features
+----------------------------------
+
+- Solvers: ``lethe-particles`` and ``lethe-fluid-particles``
+- Three-dimensional problem
+- Displays the selection of models and physical properties
+- Simulates a solid-gas cylinder shaped spouted bed
+
+
+---------------------------
+Files Used in This Example
+---------------------------
+
+Both files mentioned below are located in the example's folder (``examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed``).
+
+- Parameter file for CFD-DEM simulation of the spouted bed: ``gas-solid-spouted-bed.prm``
+- Parameter file for particle generation and packing: ``dem-packing-in-spouted-bed.prm``
+
+
+
+-----------------------
+Description of the Case
+-----------------------
+
+This example simulates the spouting of spherical particles in air. As noted in the example of `Gas-Solid Spouted Bed <../gas-solid-spouted-bed/gas-solid-spouted-beml>`_, we use ``lethe-particles``to fill the bed with particles, and ``lethe-fluid-particles`` as a solver of CFD-DEM.
+
+
+-------------------
+DEM Parameter File
+-------------------
+
+Here, we will focus only on the parts that have been modified.   
+
+Mesh
+~~~~~
+
+In this example, we are simulating a cylinder shaped spouted bed. As noted in another example, we introduce the flow through a short channel that is connected to the inlet of the bed. A schematic image is shown below;
+
+.. image:: images/geometry.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: geometry
+    :height: 10cm 
+
+The geometry of the bed was created using `Pointiwise <../../../tools/pointwise/pointowise.html>`_, and the overview of created mesh is also shown below;
+
+.. image:: images/mesh.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: mesh_ver
+    :height: 10cm
+
+In Unresolved-CFD-DEM, Cells need to have a enough volume to contain particles in it. Basically the size of cells should be least three time larger than the diameter of the particles in order to get stable calculation. So this can be expressed as follows.
```

Comment:
The cells should be able to contain several particles because of the volume-average approach only makes sense if the average is representative of the volume around it, not because it is not numerically stable. I would rephrase it. Also, I would cite the reference to this information.

Reply:
This is not exactly true. In the QCM method, the volume-averaging is carried out using a sphere which size can be defined using the cell-size. So the averaging volume can be different from the cell and should be considered to be a distinct notion.

Reply:
Uhum! My comment is not complete but I have explained it in person.